File: server/code/php/user_solution.php
<?php
function functionName ($param1, $param2, $param3) {
   //code here
   return [[1,2,3],[2,3,4]];
}


$param1 = (bool)readline();
$param2_count = intval(trim(fgets(STDIN)));
$parameter1 = [];
for ($i = 0; $i < $param2_count; $i++) {
    $param2_item = trim(fgets(STDIN));
$param2[] = $param2_item;
}

$param3_rows = intval(trim(fgets(STDIN)));
$param3_columns = intval(trim(fgets(STDIN)));
$param3 = array();
for ($i = 0; $i < $param3_rows; $i++) {
    $param3_temp = rtrim(fgets(STDIN));
    $param3[] = preg_split('/ /', $param3_temp, -1, PREG_SPLIT_NO_EMPTY);
}

$result = functionName($param1, $param2, $param3);
$resultParts = [];
foreach ($result as $row) {
    $resultsParts[] = implode(' ', $row);
}
echo implode('PHP_EOL', $resultParts);

File: server/code/java/user_solution.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;
import java.util.stream.Collectors;
import java.util.stream.IntStream;
public class Main {
   public static void main(String[] args) throws IOException {
      BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
      BufferedWriter bufferedWriter = new BufferedWriter(new OutputStreamWriter(System.out));
boolean param1 = Integer.parseInt(bufferedReader.readLine().trim()) != 0;
      int param2Count = Integer.parseInt(bufferedReader.readLine().trim());
      List<String> param2 = IntStream.range(0, param2Count).mapToObj(i -> {
         try {
            return bufferedReader.readLine().replaceAll("\\s+$", "");
         } catch (IOException ex) {
            throw new RuntimeException(ex);
         }
      })
      .map(String::trim).collect(Collectors.toList());
      int param3Rows = Integer.parseInt(bufferedReader.readLine().trim());
      int param3Columns = Integer.parseInt(bufferedReader.readLine().trim());
      List<List<String>> param3 = new ArrayList<>();
      for (int i = 0; i < param3Rows; i++) {
         List<String> param3RowItems = new ArrayList<>();
         for (int j = 0; j < param3Columns; j++) {
            String param3Item = bufferedReader.readLine().trim();
            param3RowItems.add(param3Item);
         }
         param3.add(param3RowItems);
      }

      List<List<String>> result = Result.functionName(param1, param2, param3);
      for (int i = 0; i < result.size(); i++) {
        for(int j = 0; j< result.get(i).size(); j++) {
            bufferedWriter.write(String.valueOf(result.get(i).get(j)));
            if (j != result.get(i).size() - 1) {
               bufferedWriter.write(" ");
            }
         }
         if (i != result.size() - 1) {
            bufferedWriter.write("\n");
         }
      }
bufferedWriter.newLine();
      bufferedReader.close();
      bufferedWriter.close();
   }
}
class Result {
   public static List<List<String>> functionName(boolean param1, List<String> param2, List<List<String>> param3) {
      //code here
       return param3;   }
}